src/components/ProtectedRoutes.jsx

```diff
@@ -0,0 +1,7 @@
+import { useUser } from "hooks/context/userDataContext";
```

Comment:
put context folder outside ie at App.js level

Reply:
is there any reason for that ?